faiss/python/__init__.py

```diff
@@ -194,6 +194,48 @@ def replacement_decode(self, codes):
     replace_method(the_class, 'decode', replacement_decode)
 
 
+def handle_ProductAdditiveQuantizer(the_class):
```

Comment:
This is more magic than we want for the Python interface. Please let the caller do the loop to get the subquantizer parameters.
Besides, this fails on the internal FB system with 
```

... __init__.py", line 149, in replacement_train
    assert d == self.d

AttributeError: 'ProductResidualQuantizer' object has no attribute 'd'
```